datafusion/sqllogictest/test_files/joins.slt

```diff
@@ -4009,12 +4009,12 @@ logical_plan
 09)------------Unnest: lists[__unnest_placeholder(generate_series(Int64(1),outer_ref(t1.t1_int)))|depth=1] structs[]
 10)--------------Projection: generate_series(Int64(1), CAST(outer_ref(t1.t1_int) AS Int64)) AS __unnest_placeholder(generate_series(Int64(1),outer_ref(t1.t1_int)))
 11)----------------EmptyRelation
-physical_plan_error This feature is not implemented: Physical plan does not support logical expression OuterReferenceColumn(UInt32, Column { relation: Some(Bare { table: "t1" }), name: "t1_int" })
+physical_plan_error This feature is not implemented: Physical plan does not support logical expression OuterReferenceColumn((UInt32, Column { relation: Some(Bare { table: "t1" }), name: "t1_int" }))
```

Comment:
do you know why the plan changes? I don't think the extra `()` really adds much -- can we restore the original?

Reply:
I changed it to OuterReference in latest PR, so it will use OuterReference instead of this change now, thanks!

Reply:
Next step, we may can add debug/fmt for OuterReference to restore the original behaviour.

Reply:
Restore all slt in latest PR, no diff from main branch.